openslides_backend/action/actions/meeting/import_.py

```diff
@@ -225,6 +226,8 @@ def replace_field_ids(
                     )
                     for id_ in entry[field]
                 ]
+            elif isinstance(model_field, RelationField) and isinstance(model_field, BaseTemplateField):
+                pass
```

Comment:
Why do you pass here?

Reply:
It is the case: `logo_$_id = ["webheader"]`. So I don't want to change anything. Not the fieldname and not the List.
The new code version has a `model_field.is_tempalete(field)` too and there is a test for the field.

Reply:
Ah, ok. Please remove the `isinstance(model_field, RelationField)` and replace it with `not model_field.replacement_collection` since that is the defining element of that relation (it is a structured tag, which does not have a replacement collection vs. a structured field which has one). It should still work then

Reply:
Done.